core/src/services/routing.js

```diff
@@ -405,3 +409,27 @@ export const handleRouteClick = (node, windowElem = window) => {
   const route = buildRoute(node, `/${node.pathSegment}`);
   navigateTo(route, windowElem);
 };
+
+export const getModifiedHash = s => s.newURL.split('#/')[1];
+export const getModifiedPathname = () =>
+  window.history.state.path
+    .split('/')
+    .slice(1)
+    .join('/');
+
+export const getCurrentPath = () =>
+  getConfigValue('routing.useHashRouting')
+    ? window.location.hash.replace('#', '')
```

Comment:
Can we reuse here `getPathWithoutHash` method?

Reply:
I did, but reverted again since it breaks other places throughout the app. We might want to consider refactoring later.